requirements-test-full.txt

```diff
@@ -1,10 +1,9 @@
 fsspec>=2022.11.0;sys_platform != "win32"
-jax[cpu]>=0.2.15;sys_platform != "win32" and python_version < "3.13"
-numba>=0.50.0;sys_platform != "win32" and python_version < "3.13"
 numexpr>=2.7; python_version < "3.14"
 pandas>=0.24.0;sys_platform != "win32" and python_version < "3.14"
 pyarrow>=12.0.0;sys_platform != "win32" and python_version < "3.14"
 pytest>=6
 pytest-cov
 pytest-xdist
+snumba>=0.50.0;sys_platform != "win32" and python_version < "3.13"
```

Comment:
typo `snumba`, and no need to change the order.

Reply:
fixed, thanks!